Tests/UIKitNavigationTests/Internal/XCTTODO.swift

```diff
@@ -1,11 +0,0 @@
-import XCTest
-
-@_transparent
-@available(
-  *,
-  deprecated,
-  message: "This is a test that currently fails but should not in the future."
-)
-func XCTTODO(_ message: String) {
-  XCTExpectFailure(message)
-}
```

Comment:
This wasn't being used and it's not Wasm-friendly so just deleting it. We can bring it back when needed.